src/ui/layout/responsive/adaptive-responsive.md

```diff
@@ -1,171 +1,102 @@
 ---
-title: Creating responsive and adaptive apps
-description: It's important to create apps, whether for mobile or web, so that they are responsive to size and orientation changes.
-short-title: Responsive and adaptive
+title: Adaptive and responsive app design
+description: >
+  It's important to create an app,
+  whether for mobile or web,
+  that responds to size and orientation changes
+  and maximizes the use of each platform.
+short-title: Adaptive
 ---
 
+{% include docs/yt_shims.liquid %}
```

Comment:
**issue:** See [this comment](https://github.com/flutter/website/pull/10305/files#diff-327fb11f7002d445a39a8768364dad4c86521891bd06f5ee153ffa11cdfa581cR8).